types/d3-interpolate/index.d.ts

```diff
@@ -140,6 +150,21 @@ export type ArrayInterpolator<A extends any[]> = ((t: number) => A);
  * No copy is made for performance reasons; interpolators are often part of the inner loop of animated transitions.
  */
 export function interpolateArray<A extends any[]>(a: any[], b: A): ArrayInterpolator<A>;
+/**
+ * interpolateNumberArray is called
+ */
+export function interpolateArray<T extends NumberArray>(a: NumberArray | number[], b: T): ((t: number) => T);
```

Comment:
`T extends NumberArray | number`?  I don't know anything about the functionality - it just looks like it should have the same signature as `interpolateNumberArray`.

Reply:
The `number[]` case is already covered by [the overload before this one](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48177/files/c2c4aa02a7298308b948cf29ab3936d949ced48d#diff-68305de20524395d8003b09966a09eb7R152).